contrib/python/api/skydive/websocket/client.py

```diff
@@ -19,19 +19,29 @@
 # under the License.
 #
 
-import asyncio
+try:
+    import asyncio
+except ImportError:
+    import trollius as asyncio
```

Comment:
can you update the requirements.txt

Reply:
Will do.

Reply:
Sorry forget to say something, can you update the requirements in the setup.cfg too and can you increase the minor version of the package (0.3.0) so that once your patch merged I'll push the new version to pypi